app/Resources/translations/messages.fr.yml

```diff
@@ -115,3 +115,14 @@ ton_macron.invitation_form.self_last_name: Votre nom
 ton_macron.invitation_form.self_email: Votre e-mail
 ton_macron.invitation_form.friend_email: E-mail de votre ami
 ton_macron.invitation_form.submit_send: Envoyer
+
+#
+# Event
+#
+event.invitation.form.email: Votre e-mail
+event.invitation.form.first_name: Votre prénom
+event.invitation.form.last_name: Votre nom
+event.invitation.form.message: Votre message (200 caractères max)
+event.invitation.form.guest: E-mail d'un ami
+event.invitation.form.invite: Envoyer
+event.invitation.form.invite_sent: Merci ! Votre invitation a bien été envoyée ! | Merci ! Vos %count% invitations ont bien été envoyées !
```

Comment:
Yeah, but it's not really used as it needs to enforce translation domain option everywhere.